tests/unit/test_toolchain.cpp

```diff
@@ -92,9 +104,11 @@ void TestToolchain(void)
     test_packed_enum();
 }
 
+#ifndef _WIN32
```

Comment:
I see.  Perhaps a similar model could be setup for linux, where the tests create a libunit.a that is linked to a top-level runner.exe.  But yes, understand the difference now, thanks.  Does a framework like googletest play nice with appveyor in a cross-platform way?